corona-warn-app/cwa-app-android
# FIX: 2.17 Hotfix (DEV)

App Store Crash Report:

```
de.rki.coronawarnapp.covidcertificate.signature.core.common.exception.DscValidationException: 
  at de.rki.coronawarnapp.covidcertificate.signature.core.server.DscServer.getDscList (DscServer.kt:8)
  at de.rki.coronawarnapp.covidcertificate.signature.core.server.DscServer$getDscList$1.invokeSuspend (Unknown Source:11)
```

## How to verify the fix?
- disable internet connection
- reset the App 
- open certificates tab 
- before the fix app crashes
- after the fix app should not :/
